azurelinuxagent/common/osutil/default.py

```diff
@@ -71,40 +71,47 @@ def _add_wait(wait, command):
     return command
 
 
-def _get_iptables_version_command():
+def get_iptables_version_command():
     return ["iptables", "--version"]
 
+def get_firewall_accept_dns_tcp_rule(wait, command, destination):
+    return AddFirewallRules.get_firewall_accept_dns_tcp_rule(wait, command, destination)
 
-def _get_firewall_accept_command(wait, command, destination, owner_uid):
+
+def get_firewall_accept_command(wait, command, destination, owner_uid):
     return AddFirewallRules.get_iptables_accept_command(wait, command, destination, owner_uid)
```

Comment:
Same for this

Reply:
As discussed offline all the methods in default.py will be moved to networkutil